src/tests/ie_test_utils/common_test_utils/test_constants.cpp

```diff
@@ -11,7 +11,7 @@ const char *DEVICE_CPU = "CPU";
 const char *DEVICE_GNA = "GNA";
 const char *DEVICE_GPU = "GPU";
 const char *DEVICE_BATCH = "BATCH";
-const char *DEVICE_KEEMBAY = "VPUX";
+const char *DEVICE_KEEMBAY = "NPU";
```

Comment:
We should rename this to DEVICE_NPU and move it before BATCH

Reply:
There are few other occurrences of this in OV. Functional tests in vpux-plugin need to be updated as well.